engine/src/execution_graph/logic_controllers/BatchProcessing.cpp

```diff
@@ -346,15 +346,29 @@ kstatus Projection::run() {
     CodeTimer timer;
 
     std::unique_ptr <ral::cache::CacheData> cache_data = this->input_cache()->pullCacheData();
-    while(cache_data != nullptr ){
-        std::vector<std::unique_ptr <ral::cache::CacheData> > inputs;
-        inputs.push_back(std::move(cache_data));
+    std::vector<std::string> data_names = cache_data->names();
 
-        ral::execution::executor::get_instance()->add_task(
-                std::move(inputs),
-                this->output_cache(),
-                this);
+    // When this kernel will project all the columns (with or without aliases)
+    // we want to avoid caching and decahing for this kernel
+    bool by_passing_project, by_passing_project_with_aliases;
```

Comment:
bypass or bypassing is one word, not two words. So these variables should be bypassing_project, bypassing_project_with_aliases

Reply:
byPassingProject should be bypassingProject